app/controllers/admin/users_controller.rb

```diff
@@ -0,0 +1,18 @@
+class Admin::UsersController < ApplicationController
+
+  def update
+   @user = User.find(params[:id])
+   if @user.roles.where(name: "activated").exists?
+     @user.deactivate
+    elsif
+      @user.roles.where(name: "deactivated").exists?
+      @user.activate
+    else
+    end
+    redirect_back(fallback_location: "admin/dashboard")
```

Comment:
@cmacaulay since there are no admin::user views the redirect_back sends back the info to the path that initiated the request and displays the information in the same order as before.